skopt/space.py

```diff
@@ -255,7 +258,8 @@ def rvs(self, n_samples=None, random_state=None):
 
     @property
     def transformed_size(self):
-        return len(self.categories)
+        size = len(self.categories)
+        return size if size != 2 else 1
```

Comment:
I dont understand. Surely you are adding a new `None` category right, which be counted for.